pkgs/applications/version-management/meld/fix-path.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/bin/meson.build b/bin/meson.build
+index 9a743eb8..dfd73389 100644
+--- a/bin/meson.build
++++ b/bin/meson.build
+@@ -2,6 +2,6 @@
+ configure_file(
+     input: 'meld',
+     output: 'meld',
+-    command: ['../meson_shebang_normalisation.py', '@INPUT@', '@OUTPUT@'],
++    command: [python3, join_paths(meson.current_source_dir(), '../meson_shebang_normalisation.py'), '@INPUT@', '@OUTPUT@'],
```

Comment:
We do not want this, see the meld changes in #182618.

Reply:
Shall I match that one, or just keep it in my local overlay until your PR is ready?

Reply:
Looks like the issue is actually [fixed now](https://gitlab.gnome.org/GNOME/meld/-/commit/4846b2c8e6a2602301ebd79e511f71e06d233fc5#note_1532485) so no need to match that. Just make sure to update the website. And maybe use `patchShebangs` instead of patching the build scripts:

```patch
diff --git a/pkgs/applications/version-management/meld/default.nix b/pkgs/applications/version-management/meld/default.nix
index 9c84b7bae14..121a0cdf773 100644
--- a/pkgs/applications/version-management/meld/default.nix
+++ b/pkgs/applications/version-management/meld/default.nix
@@ -27,8 +27,6 @@ python3.pkgs.buildPythonApplication rec {
     sha256 = "sha256-P8EHyY7251NY/9Kw0UyF3bSP4UoR6TmpQyL6qo6QxA0=";
   };
 
-  patches = [./fix-path.patch];
-
   nativeBuildInputs = [
     meson
     ninja
@@ -58,6 +56,10 @@ python3.pkgs.buildPythonApplication rec {
   # https://github.com/NixOS/nixpkgs/issues/56943
   strictDeps = false;
 
+  postPatch = ''
+    patchShebangs meson_shebang_normalisation.py
+  '';
+
   passthru = {
     updateScript = gnome.updateScript {
       packageName = pname;
@@ -67,7 +69,7 @@ python3.pkgs.buildPythonApplication rec {
 
   meta = with lib; {
     description = "Visual diff and merge tool";
-    homepage = "http://meldmerge.org/";
+    homepage = "https://meld.app/";
     license = licenses.gpl2Plus;
     platforms = platforms.linux ++ platforms.darwin;
     maintainers = with maintainers; [ jtojnar mimame ];
```

Reply:
That `patchShebangs` is much better. Thank you. Hopefully I didn't step on your toes with this PR. I did a search to see if there was one already but I guess draft PRs don't show up?